File: backend/Models/Order.cs
namespace MenuRestaurant.Models
{
    public class Order
    {
        public Guid Id { get; set; }
        public string? FoodName { get; set; }
        public int? Quantity { get; set; }
        public int? Price { get; set; }
        public List<Cart>? Cart { get; set; }
        public string? CartId { get; set; }
    }
}

File: frontend/src/features/order/Order.js
import React from 'react'
import { useSelector } from 'react-redux'
import { selectOrderById } from './orderApiSlice'

const Order = ({orderId}) => {
  
    const order = useSelector(state => selectOrderById(state, orderId))

  return (
   <>
 
   <tr>
    <td>{order.foodName}</td>
    <td>{order.quantity}</td>
    <td>{order.price}</td>
   </tr>
   </>
  )
}

export default Order